ansible/roles/restart-lxd/tasks/main.yml

```
@@ -2,4 +2,6 @@
   with_inventory_hostnames: ss_servers, cs_servers, cp_servers
   lxd_container:
     name: "{{item}}"
+    url: "{{ lxd_url | default('unix:/var/lib/lxd/unix.socket') }}"
+    snap_url: "{{ lxd_url | default('unix:/var/lib/lxd/unix.socket') }}"
```

```suggestion
    url: "{{ lxd_url | default(omit) }}"
```
In case lxd_url is not defined, the lxd container module should be allowed to internally discover the URL.
And there should be no need to override the snap_url (the default here breaks the playbook on Ubuntu, or when using LXD from snap).